### DISCUSSION TITLE
How do I make HTTP requests using HTTP Kernel for untrusted URL?

### DISCUSSION THREAD

**Discussion by NikiforovAll** [dotnet/interactive#3743] [Q&A]
Imagine I have an elastic search running locally on HTTPS, I would like to make HTTP requests to a HTTPS endpoint that doesn't have a trusted SSL certificate.

Here is what I see:

![image](https://github.com/user-attachments/assets/8e7aed0d-23ec-4524-8778-e21e810058b1)

**Comment by jonsequitur** [comment]
@sayedihashimi Any thoughts on this? Have we heard this question from VS users?

**Reply by sayedihashimi** [reply]
There has been suggestions to disable SSL validation. It's on the backlog but no specific date for delivery.
